src/sage/combinat/designs/orthogonal_arrays_find_recursive.pyx

```diff
@@ -732,67 +732,67 @@ cpdef find_brouwer_separable_design(int k,int n) noexcept:
         False
     """
     from sage.combinat.designs.orthogonal_arrays_build_recursive import brouwer_separable_design
-    cdef int q,x,baer_subplane_size, max_t, min_t, t,e1,e2,e3,e4
+    cdef int q, x, baer_subplane_size, max_t, min_t, t, e1, e2, e3, e4
 
-    for q in prime_powers(2,n):
-        baer_subplane_size = q**2+q+1
+    for q in prime_powers(2, n):
+        baer_subplane_size = q**2 + q + 1
         if baer_subplane_size > n:
             break
         #                       x <= q^2+1
         # <=>        n-t(q^2+q+1) <= q^2+1
         # <=>             n-q^2-1 <= t(q^2+q+1)
         # <=> (n-q^2-1)/(q^2+q+1) <= t
 
-        min_t = (n-q**2-1)/baer_subplane_size
-        max_t = min(n/baer_subplane_size,q**2-q+1)
+        min_t = (n - q**2 - 1) / baer_subplane_size
+        max_t = min(n / baer_subplane_size, q**2 - q + 1)
 
-        for t in range(min_t,max_t+1):
-            x = n - t*baer_subplane_size
-            e1 = int(x != q**2-q-t)
+        for t in range(min_t, max_t + 1):
+            x = n - t * baer_subplane_size
+            e1 = int(x != q**2 - q - t)
             e2 = int(x != 1)
             e3 = int(x != q**2)
-            e4 = int(x != t+q+1)
+            e4 = int(x != t + q + 1)
 
             # i)
             if (x == 0 and
-                is_available(k, t) and is_available(k, t+q)):
+                is_available(k, t) and is_available(k, t + q)):
```

Comment:
Sorry for the misunderstanding. I think you should add spaces to avoid having `is_available` on the same column than the return statement below.

Reply:
Do you plan to do this change or you prefer to let it for a future PR ?